prometheus_client/process_collector.py

```diff
@@ -31,6 +23,12 @@ def __init__(self, namespace='', pid=lambda: 'self', proc='/proc', registry=REGI
         except (ValueError, TypeError, AttributeError, OSError):
             pass
 
+        try:
+            self._pagesize = resource.getpagesize()
+        except ImportError:
```

Comment:
This doesn't look right, the import is elsewhere now.

Reply:
Ohh you're right. For some reason, I thought the error was triggered by `getpagesize()`.
I'll move the import back into the try block.

Reply:
Or do you prefer it still at the top, leaving the previous `_PAGESIZE` global, and rather just initialize a local variable from that?

Reply:
yeah, I might actually prefer that, much smaller diff :stuck_out_tongue: